packages/box/src/Box.spec.tsx

```diff
@@ -182,7 +183,8 @@ describe('packages/box', () => {
   });
 
   describe('packages/box/ExtendableBox', () => {
-    describe('the types work as expected', () => {
+    // eslint-disable-next-line jest/no-disabled-tests
+    describe.skip('the types work as expected', () => {
```

Comment:
👍 smart

Reply:
Were we talking about making a more concrete way and structure for type tests? Probably need a ticket to follow up on that discretely?

Reply:
👍 https://jira.mongodb.org/browse/PD-740